diff -Nur --exclude='.*~' --exclude='*~' --exclude='*.old' --exclude='.*.old' --exclude=bin --exclude=build_dir --exclude=dl --exclude=nohup.out --exclude=scripts --exclude=staging_dir --exclude=tmp --exclude=.svn attitude.armel/.config attitude.i386/.config
--- attitude.armel/.config	2014-05-02 10:10:29.000000000 +0200
+++ attitude.i386/.config	2014-05-02 10:13:23.000000000 +0200
@@ -1,5 +1,5 @@
 CONFIG_HAVE_DOT_CONFIG=y
-CONFIG_TARGET_android=y
+CONFIG_TARGET_x86=y
 CONFIG_DEVEL=y
 CONFIG_TOOLCHAINOPTS=y
 CONFIG_PACKAGE_e2fsprogs-static=m
diff -Nur --exclude='.*~' --exclude='*~' --exclude='*.old' --exclude='.*.old' --exclude=bin --exclude=build_dir --exclude=dl --exclude=nohup.out --exclude=scripts --exclude=staging_dir --exclude=tmp --exclude=.svn attitude.armel/package/openvpn-static/Makefile attitude.i386/package/openvpn-static/Makefile
--- attitude.armel/package/openvpn-static/Makefile	2014-05-02 09:47:21.000000000 +0200
+++ attitude.i386/package/openvpn-static/Makefile	2014-05-02 10:13:23.000000000 +0200
@@ -40,8 +40,8 @@
 	--disable-socks \
 	--enable-password-save \
 	--enable-small \
-	--with-ifconfig-path=/data/local/deb/armel/ifconfig \
-	--with-route-path=/data/local/deb/armel/route
+	--with-ifconfig-path=/data/local/deb/i386/ifconfig \
+	--with-route-path=/data/local/deb/i386/route
 
 TARGET_CFLAGS += -static
 
